src/compute/src/logging/mod.rs

```diff
@@ -116,3 +122,21 @@ where
             .push(Event::Progress(vec![(self.time_ms, -1)]));
     }
 }
+
+/// Parts to connect logging dataflows to the timely runtime.
+#[derive(Clone)]
+struct Plumbing<E> {
```

Comment:
I'm not sure I like the name! But I don't have a better one at hand :)

Reply:
Could we call it something like `TimelyEventQueueing` or similar? I guess that we do not want to force a future reader of the code to have to flip between the internals of `Plumbing` and where it is called to understand the code, right? If so, then the introduction of `Plumbing` would be making the code a bit harder to read than previously...

Reply:
`Plumbing` is pretty much just meant as a way to bundle up these two things, so I would actually expect you to know its internals to use it. Most of the code just passes it along though, and for that you can treat it as an opaque type.

We could make it a "smarter" type by not exposing its internals correctly by having methods like `create_logger`, `activate`, `replay`. I personally think that might be unnecessarily complex, but LMK if you think it would improve readability!

I think one issue with `Plumbing` is that it's not evocative. I'd be up with changing it to `EventQueue`, like what you suggested, just less bulky.

Reply:
I think renaming to `EventQueue` recovers most of the readability without having to create descriptive methods in `Plumbing`. So I'd be happy with that solution!